docs/appendix/troubleshooting.md

```diff
@@ -3,229 +3,102 @@ title: "Troubleshooting"
-
-<table style="border: solid black 1px;">
-<th><strong>What/Why:
-</th>
-<td> -
-</td>
-</tr>
-<tr>
-<th>Resolution
-</th>
-<td> -
-</td>
-</tr>
-</table>
-
-### Github Issues
-
-#### Error #8: Git fetch invalid
-
-<table border="solid black 1px">
-<th><strong>What/Why:
-</th>
-<td>This issue is caused by a problem with the sudo env and the user env.
-When in sudo, the RSA tokens used to connect to GitHub are removed in order to use admin credentials.
-</td>
-</tr>
-<tr>
-<th>Resolution
-</th>
-<td> -
-</td>
-</tr>
-</table>
-
-#### Error #9: Error serving web
-
-<table border="solid black 1px">
-<th><strong>What/Why:
-</th>
-<td> -
-</td>
-</tr>
-<tr>
-<th>Resolution
-</th>
-<td> -
-</td>
-</tr>
-</table>
-
-## Application and Plugin Conflicts
-
-### Mac Applications
+
+This document lists common errors encountered when working with `viam-server` and the [Viam app](https://app.viam.com), and provides simple steps to resolve them.
+While many common issues and their possible resolutions are presented here, this list is not comprehensive.
+
+If you have encountered an error that is not listed here, we'd love to hear from you on our [Community Discord](https://discord.gg/viam)!
+Please post the error message you received along with how you were able to trigger it and we'll see if we can help.
+
+## Common Installation Errors
+
+### The authenticity of host 'hostname.local' can't be established
+
+- **Description:** When following our [installation guides](/installation/), you will likely encounter this message the first time you try to make an `ssh` connection to your newly-imaged {{< glossary_tooltip term_id="board" text="board" >}}.
+   This is expected: `ssh` is advising you that it has not yet connected to this address, and prompts you for how to proceed.
+
+- **Solution:** The message will ask `Are you sure you want to continue connecting?`.
+   Type `yes` and then press the return key to continue with the connection.
+   You should receive a successful confirmation message similar to: `Warning: Permanently added 'hostname.local' to the list of known hosts.`
+   This is only required for the first `ssh` connection you make to a newly-imaged board.
+
+### ssh: connect to host hostname.local port 22: Host is down
+
+- **Description:** Your computer is not able to connect to the {{< glossary_tooltip term_id="board" text="board" >}} through `ssh`.
+
+- **Solution:** Ensure that both your computer and the board itself are connected to the internet, and verify each of the following:
+  - If you are on a Windows computer, be sure that you do not have an outgoing firewall preventing `ssh` connections over port `22`.
+  - Your `ssh` connection string should resemble the following: `ssh username@hostname.local`.
+    Be sure that you match hostname, username, and password exactly to what you initially configured when imaging your board.
+  - If you are still unable to connect, try restarting the board and trying your connection again.
```

Comment:
Great question! For `ssh` connectivity, all forms of restarting should be identical. We aren't caching credentials or temp configs in NVRAM (for which we might want 30s power-down to force a flush, for example. Common with cable modems). We do get to rely upon the system's init system (like `systemd`) to handle bringing up the `ssh` service for us, which abstracts away most of the thinking we'd have to otherwise do during boot to ensure services that depend on each other (or certain hardware to be initialized, or ...) are available, started in correct order, etc.

There is a possibility that a `restart` specifically can trigger different behavior in the service coming back online than an explicit `shutdown` or power-cable-pull (`systemd` for example allows for explicit targeting of each with: `shutdown.target` `reboot.target`, `poweroff.target`, etc.). However, `ssh` starts later in the boot process (`systemd` uses `multi-user.target`, essentially SysV init run level 3 or 4 or 5 depending on distro), where any of those "how we got to here" differences have almost certainly already been addressed.

So, from the user perspective, whatever "restart the board" means to them (pull the plug, pull the plug and wait, `sudo reboot`, `sudo shutdown -h now`) should affect the `ssh` service coming back up after the system is back up in an identical way!

Reply:
Ah cool! Thanks so much for that info!